DatabaseMigrations/2020-04-18-01-AddLogTable.cs

```diff
@@ -1,6 +1,6 @@
 using FluentMigrator;
 
-namespace simulation
+namespace simulation.DatabaseMigrations
```

Comment:
is it common to use capital letters for namespaces in .net? In Java, the classes have capitalized names but the namespaces are typically all lowercase

Reply:
They're upper camel case, for example System.Data.SqlClient:
https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient?view=netframework-4.8

Reply:
k. Ignore this then. Just looks weird to a Java guy.